nls/platform.properties

```diff
@@ -99,6 +99,7 @@ argo.placement.title = Placement
 argo.template.title = Template
 argo.template.source.title = Source
 argo.template.destination.title = Destination
+argo.placement.exception = does not exist in this environment. Choose another rule or create a new one.
```

Comment:
Can we use a substitution here? `{0} does not exist in this...`

Reply:
I was going to suggest:
`Placement {0} does not exist in this environment. Choose another rule or create a new one.`